### ISSUE TITLE
Please list the URL of the public GPG key

### ISSUE THREAD

**Issue by sid-the-sloth** [portfolio-performance/portfolio#4342]
**Describe the bug**
PP publishes a binary archive and a signature file (.asc) for each platform. One is supposed to verify the integrity of the binary using the asc file, but the location of the GPG public key is _unknown_. Please publish the GPG public key location in the readme file.

**To Reproduce**
1. download `PortfolioPerformance-0.71.2-linux.gtk.x86_64.tar.gz`
2. download `PortfolioPerformance-0.71.2-linux.gtk.x86_64.tar.gz.asc`
3. in terminal: `gpg --verify PortfolioPerformance-0.71.2-linux.gtk.x86_64.tar.gz.asc PortfolioPerformance-0.71.2-linux.gtk.x86_64.tar.gz`
4. output from above: 
```
gpg: Signature made Thu 03 Oct 2024 03:52:29 PM EDT
gpg:                using RSA key E46E6F8FF02E4C83569084589239277F560C95AC
gpg: Can't check signature: No public key
```

**Expected behavior**
Somebody should publish the location of the GPG public key (preferably in the readme file) so users can import that public key.

**Desktop (please complete the following information):**
 - OS: Debian GNU/Linux 12 (bookworm)

**Comment by buchen** [comment]
good point. the key is published on GPG keyserver and Ubuntu key server, but could also add it to the readme.

https://keyserver.ubuntu.com/pks/lookup?search=andreas.buchen%40gmail.com&fingerprint=on&op=index
